patches/tModLoader/Terraria/ModLoader/ModContent.cs

```diff
@@ -341,6 +341,7 @@ internal static void Load(CancellationToken token)
 		ContentSamples.RebuildItemCreativeSortingIDsAfterRecipesAreSetUp();
 		ItemSorting.SetupWhiteLists();
 		LocalizationLoader.UpdateLocalizationFiles();
+		LanguageManager.Instance.SetupFileWatchers();
```

Comment:
Please move all the file watcher code into `LocalizationLoader`, it's a more natural place.

Then make a single `FinishSetup` method which calls `UpdateLocalizationFiles` and `SetupFileWatchers`
and a single `Unload` method which calls `LanguageManager.Instance.UnloadModdedEntries` and `UnloadFileWatchers`